dashboard/app/models/ability.rb

```diff
@@ -1,53 +1,47 @@
 class Ability
   include CanCan::Ability
 
+  # Define abilities for the passed in user here. For more information, see the
+  # wiki at https://github.com/ryanb/cancan/wiki/Defining-Abilities.
   def initialize(user)
-    # Define abilities for the passed in user here. For example:
-    #
-    user ||= User.new # guest user (not logged in)
-    if user.admin?
-      can :manage, :all
+    user ||= User.new
 
-      # Only custom levels are editable
-      cannot [:update, :destroy], Level do |level|
-        !level.custom?
-      end
-    else
-      can :read, :all
-      cannot :read, [
-        Script, # see override below
-        ScriptLevel, # see override below
-        PrizeProvider,
-        Prize,
-        TeacherPrize,
-        TeacherBonusPrize,
-        LevelSourceHint,
-        FrequentUnsuccessfulLevelSource,
-        :reports,
-        User,
-        Follower,
-        PeerReview,
-        # Ops models
-        District,
-        Workshop,
-        Cohort,
-        WorkshopAttendance,
-        # PLC Stuff
-        Plc::Course,
-        Plc::LearningModule,
-        Plc::Task,
-        Plc::UserCourseEnrollment,
-        Plc::CourseUnit,
-        # PD models
-        Pd::Workshop,
-        Pd::Attendance,
-        Pd::DistrictPaymentTerm,
-        Pd::DistrictReport,
-        Pd::WorkshopOrganizerReport,
-        Pd::TeacherProgressReport,
-        Pd::CourseFacilitator
-      ]
-    end
+    # Abilities for all users, signed in or not signed in.
+    can :read, :all
+    cannot :read, [
+      Script, # see override below
+      ScriptLevel, # see override below
+      PrizeProvider,
+      Prize,
+      TeacherPrize,
+      TeacherBonusPrize,
+      LevelSourceHint,
+      FrequentUnsuccessfulLevelSource,
+      :reports,
+      User,
+      UserPermission,
+      Follower,
+      PeerReview,
+      # Ops models
+      District,
+      Workshop,
+      Cohort,
+      WorkshopAttendance,
+      # PLC Stuff
+      Plc::Course,
+      Plc::LearningModule,
+      Plc::Task,
+      Plc::UserCourseEnrollment,
+      Plc::CourseUnit,
+      # PD models
+      Pd::Workshop,
+      Pd::Attendance,
+      Pd::DistrictPaymentTerm,
+      Pd::DistrictReport,
+      Pd::WorkshopOrganizerReport,
+      Pd::TeacherProgressReport,
+      Pd::CourseFacilitator
+    ]
 
     if user.id
```

Comment:
while you're refactoring, `user.persisted?` might be a more meaningful way to say this